presto-main/src/main/java/com/facebook/presto/sql/planner/DependencyExtractor.java

```diff
@@ -36,8 +37,13 @@ private DependencyExtractor() {}
 
     public static Set<Symbol> extractUnique(PlanNode node)
     {
+        return extractUnique(node, n -> n);
+    }
+
+    public static Set<Symbol> extractUnique(PlanNode node, Lookup lookup)
```

Comment:
You can still use `ExpressionExtractor.extractExpressionsNonRecursive` if needed but there was not equivalent `extractUnique` before nor it exists now.